spark/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -278,16 +278,16 @@ trait DeltaErrorsSuiteBase
         Some("NOT NULL constraint violated for column: col1.\n"))
     }
     {
-      val expr = CatalystSqlParser.parseExpression("concat(\"hello \", \"world\")")
+      val expr = UnresolvedAttribute("col")
       val e = intercept[DeltaInvariantViolationException] {
         throw DeltaInvariantViolationException(
           Constraints.Check(CharVarcharConstraint.INVARIANT_NAME,
             LessThanOrEqual(Length(expr), Literal(5))),
-          Map.empty[String, Any])
+          Map("col" -> "Hello World"))
       }
       checkErrorMessage(e, Some("DELTA_EXCEED_CHAR_VARCHAR_LIMIT"), Some("22001"),
```

Comment:
Can you migrate this to `checkError`?

Reply:
It seems like the goal of this suite is to actually test the final error messages and error codes. This wouldn't be possible with `checkError` (as its purpose to be able to write tests without having to match the error message).